agent-inject/agent/annotations.go

```diff
@@ -550,6 +558,22 @@ func (a *Agent) agentCacheEnable() (bool, error) {
 	return strconv.ParseBool(raw)
 }
 
+func (a *Agent) agentCachePersist() bool {
```

Comment:
Nit: the caller is `*Agent` in here so you could rename this to `cachePersist` (same with the func below).

Reply:
Yep, done in 00f91529016d93f037dde237a0897e98500cab96